onnx/defs/tensor/defs.cc

```diff
@@ -1634,19 +1632,12 @@ ONNX_OPERATOR_SET_SCHEMA(
             axes = ParseData<int64_t>(axes_proto);
```

Comment:
But this may not be a compile-time constant. Need to handle the case where axes_proto is null

Reply:
I did see there is a `axes_proto == nullptr` condition here... Could you please elaborate your case? Thank you!

Reply:
Hmmm. I am confused. Not sure what code I was looking at! Please ignore this comment!